docs/howto/edit-in-protege.md

```diff
@@ -0,0 +1,94 @@
+
+To add a dbxref to the definition:
+
+-   Click the @ symbol next to the definition
+-   Click the + button next in the pop-up window
+-   Scroll up on the left hand side until you find 'database_cross_reference', and click it
+-   Add the PMID in the editing box (PMID:25527564). _Note: the PMID should not have any spaces)
+-   Click OK
+-   Add the additional dbxref: GARD:0013029
+-   The dbxrefs should appear as below.
+
+![](https://lh6.googleusercontent.com/l589uvv3OKKxrabrqKQdL-NF6PfKi_mSfaz-xk--59WtSD15VOy9CQVZXdE0SHl6ZA761zv9G0UULHF5EKRfMToX2F0kqrwuGbjdnzVV3JRRJbb2l40UjOLeXi-7aM_TBkCSkN3L)
+
+![](https://lh6.googleusercontent.com/aW3quN013aSDfyFXpn-_prKrn0TN7eMzodwK4HdryZ_Zbjade5xZWnFCVt8flkRqIbMy5eT5lKzFEimuGNgJ3YYYybI5rgdcmVWUzzfdwFeXjJSFBpNjqgv27kZVPiazcMiZABn1)
+
+#### Add Synonyms and Database cross reference
+
+1.  Add synonyms 
+1.  Click the add annotations button
+1.  Add the synonyms as 'has_exact_synonym' (note: use appropriate synonym annotation) 
+1.  Synonyms should have a reference to it
+1.  Click the @ symbol next to the synonym
+1.  Click the + button
+1.  Select `database_cross_reference` on the left panel and add your reference to the Literal tab on the right hand side
+
+<a name="class-description"></a> 
+### The Class description view
+
+We have seen how to add sub/superclasses and annotate the class hierarchy. Another way to do the same thing is via the Class description view. When an OWL class is selected in the entities view, the right-hand side of the tab shows the class description panel. If we select the 'vertebral column disease' class, we see in the class description view that this class is a "SubClass Of" (= has a SuperClass) the 'musculoskeletal system disease' class. Using the (+) button beside "SubClass Of" we could add another superclass to the 'skeletal system disease' class.
+
+Note the Anonymous Ancestors. These are superclasses that are inherited from the parents. If you hover over the Subclass Of (Anonymous Ancestor) you can see the parent that the class inherited the superclass from.
+
+![](https://lh4.googleusercontent.com/hC3R3tw3S5eVNLc70iCN0lrtj9rD_gIPUBxberpw4gSRRR6xct1Xv5dHYSfXPchpYvpGMhIgGnQQ18dl6pWicyClpL-GGyi_JjkeSKOetm4hleSfA-kxu6ww6v-3q-NOLj3xhd7m)
+
+#### Revising a superclass:
+
+1.  If you want to revise the superclass, click the 'o' symbol next to the superclass and replace the text. Try to revise 'musculoskeletal system disease' to  'disease by anatomical system'.
+
+2.  If you want to delete a superclass, click the 'x' button next to the superclass. Delete the 'disease by anatomical system' superclass.
+
+Close this window without saving.
+
+Save your work.
+
+### Make a Pull Request
+
+1.  Click: Create Pull Request in GitHub Desktop
+
+2.  This will automatically open GitHub Desktop 
```

Comment:
Should lines 84 and 86 both reference GitHub Desktop?

Reply:
will see if we have docs for it and link if we do